nixos/modules/services/matrix/draupnir.md

```diff
@@ -0,0 +1,86 @@
+# Draupnir (Matrix Moderation Bot) {#module-services-draupnir}
```

Comment:
You need to add an entry in redirects.json for each section with ID
- module-services-draupnir
- module-services-draupnir-setup
- module-services-draupnir-matrix-synapse-antispam
- module-services-draupnir-setup-ems